src/widgets/ot_drawer.eliom

```diff
@@ -152,7 +152,7 @@ let%shared drawer
 
   let _ = [%client (
     let%lwt () = Ot_nodeready.nodeready (To_dom.of_element ~%d) in
-    (Eliom_client.onunload @@ fun () ->
+    (Eliom_client.Page_status.oninactive @@ fun () ->
```

Comment:
Shall we remove the `top: xxx` from the body's style and scroll back the body as well?